packages/pigment-css-react/README.md

```diff
@@ -663,6 +663,93 @@ declare module '@pigment-css/react/theme' {
 }
 ```
 
+## Right-to-left support
+
+Pigment CSS offers a built-in mechanism to support right-to-left (RTL) or left-to-right (LTR) CSS automatically. To turn it on, start by updating your bundler config:
+
+### Next.js
+
+```js
+const { withPigment } = require('@pigment-css/nextjs-plugin');
+
+// ...
+module.exports = withPigment(nextConfig, {
+  theme: yourCustomTheme,
+  // CSS output option
+  css: {
+    // Specify your default direction
+    defaultDirection: 'ltr',
```

Comment:
Here, it's showcasing how to get the output for rtl as well when your default styles are authored in `ltr`. That's why the `defaultDirection: 'ltr'`.

Reply:
So the two values are always required `defaultDirection` and `generateForBothDir`. This is to handle the case when they'd want to output css for the default direction that they are writing in and `generateForBothDir: true` means that also output the css for the non-default direction.
I guess it's confusing. There won't be a case where you are only specifying `defaultDirection` and not `generateForBothDir`

Reply:
> and the developer wants to have access to both sets of styles

Would this be the case if I want, say, to have a toggle to switch between them? Like, I'd have both available/generated. Maybe we should also be specific about the use case instead of more abstract?

Reply:
@danilo-leal Correct. The combination of the two options allow you to do that. You can use a toggle on the page to switch direction and it'll happen instantly because the css is already present.

@samuelsycamore The basic rtl configuration is no configuration. This is based on the assumption that the css you are writing already tackles that. Like if you'd normally use margin-left in ltr, you'd write margin-right instead when your default thought process is for rtl.

Reply:
Would it make sense to add a separate section for the `generateForBothDir` prop? We could be specific about what's the main use case for it. I was thinking of something along these lines:

> **Generate CSS for both directions**
> Turn the `generateForBothDir` prop to true if you want to generate CSS for both directions. This would allow, for example, your user to toggle the app's direction, which would happen instantly, given the CSS is already available.

Reply:
Okay, just pushed a new suggestion about this — let me know if that makes sense!